ui/src/app/pages/login/account-creation/account-creation-survey.tsx

```diff
@@ -138,70 +140,75 @@ export class AccountCreationSurvey extends React.Component<AccountCreationSurvey
       </ListPageHeader>
 
       {/*Race section*/}
-      <Section header='1. Race'>
+      <Section header='Race'>
         <div style={{display: 'flex', justifyContent: 'flex-start', flexWrap: 'wrap'}}>
           {AccountCreationOptions.race.map((race) => {
-            return this.createOptionCheckbox(race.label, 'race', race.value);
+            return this.createOptionCheckbox(race.label, 'race', race.value, race.value.toString());
           })}
         </div>
       </Section>
 
       {/*Ethnicity section*/}
-      <DropDownSection header='2. Ethnicity' options={AccountCreationOptions.ethnicity}
+      <DropDownSection header='Ethnicity' options={AccountCreationOptions.ethnicity}
                        value={demographicSurvey.ethnicity}
                        onChange={(e) => this.updateDemographicAttribute('ethnicity', e)}/>
-
-      {/*Gender section*/}
-      <Section header='3. Gender'>
-        <FlexRow style={{flexWrap: 'wrap'}}>
-          {AccountCreationOptions.gender.map((gender) => {
-            return this.createOptionCheckbox(gender.label, 'gender', gender.value);
-          })}
-        </FlexRow>
+      <Section header='Do you identify as lesbian, gay, bisexual, transgender, queer (LGBTQ),
+or another sexual and/or gender minority?'>
+        <div style={{paddingTop: '0.5rem'}}>
+          <RadioButton onChange={
+            (e) => this.toggleIdentifiesAsLgbtq(true)}
+                       checked={demographicSurvey.identifiesAsLgbtq}
+                       style={{marginRight: '0.5rem'}}/>
+          <label style={{paddingRight: '3rem', color: colors.primary}}>
+            Yes
```

Comment:
Nit: can you make the <label> fit on one line to be consistent with the "no" label? (If it got auto-wrapped, that's fine, but by eye it looks like it might fit.

Reply:
Probably! Let me try.